lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -199,16 +200,29 @@ public void execute(Publisher publisher) throws ConnectorException {
         publisher.publish(doc);
       }
 
-      // close all results
-      rs.close();
-      for (ResultSet ors : otherResults) {
-        ors.close();
-      }
       // the post sql.
       runSql(connection, postSql);

```

Comment:
I moved the runSql call to after the closing of the ResultSets, but I didn't surround it with a try catch because the SQLException is already caught in runSql. Let me know if I'm misunderstanding it.